config/config.go

```diff
@@ -24,4 +24,10 @@ const (
 	// ClusterDeploymentManagedLabel is the label the clusterdeployment will have that determines
 	// if the cluster is OSD (managed) or not
 	ClusterDeploymentManagedLabel string = "api.openshift.com/managed"
+	// NoAlertsLabel is the label the clusterdeployment will have if the cluster is not meant
+	// to be generating alerts (including DMS)
+	NoAlertsLabel string = "ext-managed.openshift.io/noalerts"
+	// LegacyNoAlertsLabel is the legacy label the clusterdeployment will have if the cluster is
+	// not meant to be generating alerts (including DMS)
+	LegacyNoAlertsLabel string = "api.openshift.com/noalerts"
```

Comment:
These are used only by tests and there is on logic that actually turns off DMS when they are present.  This logic was removed and moved to DSMI config for our envs (ironically this is why we noticed this issue needed to be fixed).

Reply:
Agree on this point - I've removed this now. (see my comment on the topic of the confusing test)